mri_seghead/mri_seghead.cpp

```diff
@@ -47,9 +47,9 @@ static void print_help(void) ;
 static void print_version(void) ;
 static void argnerr(char *option, int n);
 static void dump_options(FILE *fp);
-static int  isflag(char *flag);
+static int  isflag(const char *flag);
```

Comment:
I saw this routine in a number of files as well. Statically declared and (apparently) copied between them.